style/amenity-points.mss

```diff
@@ -3028,6 +3028,38 @@
       }
     }
   }
+
+  [feature = 'man_made_goods_conveyor'] {
+    [zoom >= 12] {
+      line/line-width: 1;
+      line/line-join: round;
+      line/line-cap: round;
+      line/line-color: #999999;
+
+      dash/line-width: 3.5;
+      dash/line-join: round;
+      dash/line-color: #999999;
+      dash/line-dasharray: 12,6;
+      dash/line-clip: false;
+      [zoom >= 17] {
+        text-name: "[name]";
```

Comment:
It would be better to render the name in the `text-line` layers. See PR #4107 for an example.

Reply:
fixed